py/torch_tensorrt/dynamo/_exporter.py

```diff
@@ -16,6 +17,28 @@
     OutputSpec,
     TensorArgument,
 )
+from torch_tensorrt.dynamo.runtime._TorchTensorRTModule import ENGINE_IDX, NAME_IDX
+
+
+def cross_save_for_windows(
+    gm: torch.fx.GraphModule,
+    file_path: str,
+) -> None:
+    gm = copy.deepcopy(gm)
+
+    # Inline TensorRT submodules for windows
+    inline_trt_modules_for_windows(gm)
```

Comment:
I think define `cross_compile` argument in `inline_trt_modules` call and handle the corresponding cross compile logic within it. This way, we don't need to copy the code twice and it's easier to maintain.

Reply:
So this save function could effectively be something like 
```py
def save_cross_compiled_exported_program():
        ep = export(gm, cross_compile=True)
        torch.export.save(exp_program, file_path)

def export(gm: torch.fx.GraphModule, cross_compile: bool = False):
        patched_module = transform(gm, cross_compile=cross_compile)
        exp_program = create_trt_exp_program(patched_module)
```